api/serverConfig.ts

```diff
@@ -0,0 +1,12 @@
+import * as sdk from 'node-appwrite';
+
+const URL = process.env.NEXT_PUBLIC_APPWRITE_API_URL as string;
+const PROJECT_ID = process.env.NEXT_PUBLIC_APPWRITE_PROJECT_ID as string;
+const API_KEY = process.env.APPWRITE_API_KEY as string;
+
+const client = new sdk.Client()
+  .setEndpoint(URL)
+  .setProject(PROJECT_ID)
+  .setKey(API_KEY);
+
+export const users = new sdk.Users(client);
```

Comment:
Should we delete the current config file since ultimately, we want to use Server SDK and not Web?

Reply:
No. We need to keep the current config file as that is the config for the web SDK.

We will be using both web and server SDKs.